docs/csharp/language-reference/operators/null-parameter-check.md

```diff
@@ -0,0 +1,81 @@
+---
+title: "!! (null validation) operator - C# reference"
+description: "Learn about simplified null-parameter checks. This operator instructs the compiler to add runtime checks that the argument used for a parameter isn't null."
+ms.date: 03/30/2022
+helpviewer_keywords:
```

Comment:
Can you add f1 keywords?

Reply:
Yes! Where do I find the keys for those?

Reply:
@BillWagner They are not added to roslyn yet. So Roslyn can use any thing you choose. Though `!!_CSharpKeyword` will be consistent with the rest.